modules/aws/cac/main.tf

```diff
@@ -10,20 +10,34 @@ locals {
   startup_script = "cac-startup.sh"
 }
 
-data "template_file" "startup-script" {
-  template = file("${path.module}/${local.startup_script}.tmpl")
+resource "aws_s3_bucket_object" "cac-startup-script" {
+  count = tonumber(var.instance_count) == 0 ? 0 : 1
+
+  key     = local.startup_script
+  bucket  = var.bucket_name
+  content = templatefile(
+    "${path.module}/${local.startup_script}.tmpl",
+    {
+      cam_url                  = var.cam_url,
+      cac_installer_url        = var.cac_installer_url,
+      cac_token                = var.cac_token,
+      pcoip_registration_code  = var.pcoip_registration_code,
+
+      domain_controller_ip     = var.domain_controller_ip,
+      domain_name              = var.domain_name,
+      domain_group             = var.domain_group,
+      service_account_username = var.service_account_username,
+      service_account_password = var.service_account_password,
```

Comment:
Should we make the variable names the same between gcp and aws? I can do a separate commit to change to ad_service_account_username and ad_service_account_password

Reply:
Could you create a new patch to update those variable names like in GCP please?